Composer/packages/client/src/pages/publish/Publish.tsx

```diff
@@ -254,6 +255,9 @@ const Publish: React.FC<RouteComponentProps<{ projectId: string; targetName?: st
         token = await AuthClient.getARMTokenForTenant(tenant);
       }
     }
+    // get keyvault token
+    const result = await AuthClient.getAccessToken(vaultScopes);
```

Comment:
This should not be done for every type of publish. Publishing to PVA should not require a call to get a KeyVault access token

Reply:
This code is only for testing, sorry we forget to remove these code. Currently, we implement the get keyvault token in azure BotServiceProvider in server.

Reply:
I just remove it